instrumentation/rabbitmq-2.7/javaagent/src/test/groovy/RabbitMqTest.groovy

```diff
@@ -340,7 +340,7 @@ class RabbitMqTest extends AgentInstrumentationSpecification implements WithRabb
 
       attributes {
         "${SemanticAttributes.NET_PEER_NAME.key}" { it == null || it instanceof String }
-        "${SemanticAttributes.NET_PEER_IP.key}" { "127.0.0.1" }
+        "${SemanticAttributes.NET_PEER_IP.key}" { it == null || it == rabbitMqContainer.containerIpAddress }
```

Comment:
Can it be null?

Reply:
Yes, apparently some of these tests expect null here. (just for the record, `{ "127.0.0.1" }` was treated as a predicate that always returns "truthy" value so it didn't check the IP address at all 🙈 )